Dockerfile

```diff
@@ -22,24 +21,25 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 WORKDIR /root
 
+ENV CONDA_VERSION 4.6.14
+COPY docker/conda_env.yml /root/conda_env.yml
 RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh && \
         /bin/bash ~/miniconda.sh -b -p /opt/conda && \
         rm ~/miniconda.sh && \
         ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh && \
         echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc && \
-        echo "conda activate patientMatcher" >> ~/.bashrc
+        echo "conda activate patientMatcher" >> ~/.bashrc && \
+        source /opt/conda/etc/profile.d/conda.sh && \
+        conda install -y conda=$CONDA_VERSION && \
+        conda env create -f /root/conda_env.yml && \
+        conda clean -tipsy
 
 COPY . /opt/patientMatcher
-
 WORKDIR /opt/patientMatcher
-
 RUN source /opt/conda/etc/profile.d/conda.sh && \
-    conda install -y conda=4.6.* && \
-    conda env create -f /opt/patientMatcher/docker/conda_env.yml && \
-    conda clean -tipsy && \
     conda activate patientMatcher && \
     pip install git+https://github.com/Clinical-Genomics/patient-similarity && \
     pip install -e .
 
 ENTRYPOINT [ "docker/entrypoint.sh" ]
-CMD [ "pmatcher", "run", "-h", "0.0.0.0" ]
+CMD [ "pmatcher", "run", "-h", "0.0.0.0", "-p", "9020" ]
```

Comment:
That's the same port I think I described in the instructions, so yes, let's have this one! :)